app/(auth)/signin.js

```diff
@@ -61,6 +84,9 @@ export default function SignIn() {
     }
     try {
       await sendPasswordResetEmail(getAuth(), email);
+      const date = Date.now() + 60000;
+      await AsyncStorage.setItem("timeNextPasswordReset", date.toString());
+      setTimeIntervalPassword(60);
```

Comment:
60 seconds seems a bit excessive IMO, maybe just 30??
```suggestion
      setTimeIntervalPassword(30);
```